pkg/operator/ceph/disruption/clusterdisruption/mon.go

```diff
@@ -61,9 +64,29 @@ func (r *ReconcileClusterDisruption) reconcileMonPDB(cephCluster *cephv1.CephClu
 			MinAvailable: &intstr.IntOrString{IntVal: minAvailable},
 		},
 	}
-	err := r.reconcileStaticPDB(pdbRequest, pdb)
```

Comment:
How about making these code changes inside `reconcileStaticPDB()`? Seems like we would want the update pattern other places as well, right? (or at least it would be a no-op if the minAvailable never changes)

Reply:
moved changes to `reconcileStaticPDB()` method.